blockscout-ens/graph-node/subgraphs/zns-ink-subgraph/tests/zns-registry-utils.ts

```
@@ -0,0 +1,295 @@
+import { newMockEvent } from "matchstick-as"
+import { ethereum, Address, BigInt, Bytes } from "@graphprotocol/graph-ts"
```

Biome (1.9.4)

[error] Do not shadow the global "BigInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)